src/video_core/texture_cache/texture_cache.h

```diff
@@ -158,9 +162,8 @@ class TextureCache {
     /// Blit an image with the given parameters
     void BlitImage(const Tegra::Engines::Fermi2D::Surface& dst,
                    const Tegra::Engines::Fermi2D::Surface& src,
-                   const Tegra::Engines::Fermi2D::Config& copy,
-                   std::optional<Region2D> src_region_override = {},
-                   std::optional<Region2D> dst_region_override = {});
+                   const int offset,
```

Comment:
```suggestion
                   int offset,
```

Given top-level const doesn't do anythin in a declaration for pass-by-value.